doc/user-guide.md

```diff
@@ -85,11 +85,14 @@ type MemcachedSpec struct {
 	Size int32 `json:"size"`
 }
 type MemcachedStatus struct {
-	// Nodes are the names of the memcached pods
+	// Nodes are the names of the memcached pods 
+	// +listType=set  
 	Nodes []string `json:"nodes"`
 }
 ```
 
+**NOTE** To know more about `+listType=set` see : https://godoc.org/k8s.io/kube-openapi/pkg/idl 
```

Comment:
```suggestion
**NOTE:** Comment directives, such as +listType=set, are necessary for certain situations to avoid API rule violations when generating OpenAPI files. See https://godoc.org/k8s.io/kube-openapi/pkg/idl learn more.
```